notification/docs/IntegrationGuide.md

```diff
@@ -58,6 +58,14 @@ Replace all `XXX` values and execute:
 docker run -e CTP_PROJECT_KEY=XXX -e CTP_CLIENT_ID=XXX -e CTP_CLIENT_SECRET=XXX ctp-adyen-integration-notification:XXX
 ```
 
+### Deployment using AWS Lambda
```

Comment:
Could you execute `doctoc` to add this section to the header?
Should be as easy as
```
> npm install -g doctoc
> doctoc .
```

Reply:
Done